Azure/azure-cli
# [Compute] `az vm update`: Add `--host` argument and `--host-group` argument to support assign an existing VM to a specific ADH

Description

[issue link](https://github.com/Azure/azure-cli/issues/19622)

Add --host argument to az vm update.
Add --host-group argument to az vm update.

Testing Guide

az vm deallocate -n {vm-name} -g {rg}

az vm update -n {vm-name} -g {rg} --host {host_id}  
or  
az vm update -n {vm-name} -g {rg} --host-group {host-group}')

az vm start -n {vm-name} -g {rg}')

---

This checklist is used to make sure that common guidelines for a pull request are followed.
